pysatModels/__init__.py

```diff
@@ -26,4 +26,4 @@
 
 # Define a logger object to allow easier log handling
 logging.raiseExceptions = False
-logger = logging.getLogger('pysat.pysatModels')
+logger = logging.getLogger('pysatModels')
```

Comment:
Ok, super confused on how changing the logging name fixed things before!

Reply:
> Looks good to me though I do have a note about the change in logging.

This is why I needed your review, didn't want to break things.  But I don't understand why the logger name needs to start with "pysat."

Reply:
My hypothesis is the logging module is setting the formatter and such for pysatModels to null. It may still be talking but it doesn't get rendered though. Starting the name with pysat associates the pysatModels logger with the pysat logger and keeps them on the same page.

Reply:
Ok, I cleaned up the other way you implemented things so that we don't carry around unneeded variables.  The old example you used in #96 works now.